File: 2706. Buy Two Chocolates
/Solution.py
class Solution:
    def buyChoco(self, prices: List[int], money: int) -> int:
        first, second = 101, 101
        for p in prices:
            if p < first:
                second, first = first, p
            elif p < second:
                second = p
        return money if (money < first + second) else money - (first + second)

File: 2706. Buy Two Chocolates
/Solution.java
class Solution {
    public int buyChoco(int[] prices, int money) {
        int first = 101, second = 101;
        for (int p : prices) {
            if (p < first) {
                second = first;
                first = p;
            } else if (p < second) {
                second = p;
            }
        }
        return (money < first + second) ? money : money - (first + second);
    }
}
